recipes/aws-c-http/all/conanfile.py

```diff
@@ -26,20 +24,20 @@ class AwsCHttp(ConanFile):
         "fPIC": True,
     }
 
-    def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
-
-    def configure(self):
-        if self.options.shared:
-            self.options.rm_safe("fPIC")
-        self.settings.rm_safe("compiler.cppstd")
-        self.settings.rm_safe("compiler.libcxx")
+    implements = ["auto_shared_fpic"]
+    languages = "C"
 
     def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
+        if self.version == "0.9.3":
+            self.requires("aws-c-common/0.11.0", transitive_headers=True, transitive_libs=True)
+            self.requires("aws-c-compression/0.3.1")
+            # Upstream uses this even if it does not explicitly state it in the CMakeLists
+            # Maybe expecting the headers to be there transitively?
+            self.requires("aws-c-cal/0.8.3")
```

Comment:
Removing this would cause missing includes, it's directly used, so either:
* Upstream CMakeLists would need to reflect this
* Or they expect one of the other dependencies to be providing this transitevely, but none in CCI have transitive traits for this set to true